CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.7.4] - 2023-26-11
```

Comment:
And made available for use with Python 3.11 and Python 3.12?

Reply:
That part was always true: it's just arbitary tagging on pypi that was out of date because Poetry didn't know about them to generate them with it's build command. As a rule Python 3.x isn't going to break backwards compatability, adopting features from features only available in 3.9 would break 3.8 compatability but we haven't done that. Means we can't use `match` or nicer typehints, but yeah - that'd be a 2.x.x release to break Python interpreter compatability.

Reply:
Thanks for explaining. I did just check now using a container for Python 3.11 and its pip lists botocove versions.

```console
$ docker run -it --rm cimg/python:3.11 pip index versions botocove | grep "Available versions"
Available versions: 1.7.4, 1.7.3, 1.7.2, 1.7.1, 1.6.4, 1.6.3, 1.6.2, 1.6.1, 1.6.0, 1.5.2, 1.5.1, 1.5.0, 1.4.1, 1.4.0, 1.3.1, 1.3.0, 1.2.0, 1.1.0, 1.0.0, 0.1.3, 0.1.2, 0.1.1, 0.1.0
```

Python 3.7's pip on the other hand lists just one ancient version.

```console
$ docker run -it --rm cimg/python:3.7 pip index versions botocove | grep "Available versions"
Available versions: 0.1.1
```